packages/dma-library/src/strategies/ajna/multiply/close.ts

```diff
@@ -73,13 +73,22 @@ export const closeMultiply: AjnaCloseStrategy = async (args, dependencies) => {
     postSwapFee.times(ONE.plus(FEE_ESTIMATE_INFLATOR)).integerValue(BigNumber.ROUND_DOWN),
   )
 
+  // Validation
+  const errors = [
+    // Add as required...
+  ]
+
+  const warnings = [
+    // Add as required..
+  ]
+
```

Comment:
I guess we didn't consider the liquidation case, but let's refine that and we will add additional checks in separate story.

Reply:
OK separate story it is.